index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200&display=swap"
+    rel="stylesheet">
+</head>
+<body>
+  <section class="container">
+    <nav class="navbar d-flex justify-content-between">
+      <div class="logo hover">
+        My Logo
+      </div>
+      <div class="menu hover">
+        <img src="img/Union.png" alt="Menu button">
+      </div>
+    </nav>
+    <main class="main d-flex">
+        <div class="intro">
+            I'm Alexandra. Glad to see you!
+        </div>
+        <div class="intro-sub">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </div>
+        <div class="socials d-flex">
+          <div class="socials-header">Let's Connect</div>
+          <div class="socials-icons d-flex justify-content-between">
+            <div class="twitter hover">
+              <img src="img/svg/Vector.svg" alt="Twiter icon">
+            </div>
+            <div class="linkedin hover">
+              <img src="img/svg/Linkedin icon.svg" alt="Linkedin icon">
+            </div>
+            <div class="medium hover">
+              <img src="img/svg/M.svg" alt="Medium icon">
+            </div>
+            <div class="github hover">
+              <img src="img/svg/github.svg" alt="Github icon">
+            </div>
+            <div class="vector hover">
+              <img src="img/svg/vector-icon.svg" alt="Vector icon">
+            </div>
+          </div>
+        </div>
+      </main>
```

Comment:
- [ ] Excellent job designing the HTML layout for your headline section 🚀. However, please kindly make sure that you use the `main` semantic tag to wrap all your sections, currently, you are using the `main` tag as the main wraper for your headline but it will be a good idea to use a `section` tag for this instead and then include that section inside your `main` tag 🤓.
This is a small example of a semantic HTML design 👇🏽, you can read more information about it [here](https://www.w3schools.com/html/html5_semantic_elements.asp)

```
<html>
    <head>
        <title>Portfolio</title>
    </head>
    <body>
        <header>
            Here goes logo, navigation, etc.
        </header>
        <main>
            <section>
                Here you can include your headline
            </section>
            <section>
                Here you can include My recent works
            </section>
        </main>
        <footer>
            Footer information, links, etc.
        </footer>
    </body>
</html>
```